gitweb/gitweb.perl

```diff
@@ -7790,7 +7790,7 @@ sub git_snapshot {
 		my $error = $@ || 'Unknown failure';
 		print $cgi->header(-status => '500 Execute git-archive failed');
 		if ($DEBUG) {
-			printf STDERR "Execute git-archive failed with exit-code $gitcode (" + $error + ")";
+			printf STDERR "Execute git-archive failed with exit-code $gitcode ($error)";
```

Comment:
You will want to squash the commit "gitweb.perl : typo fix in debug printout" into the commit that introduced the problem.